scss/_variables.scss

```diff
@@ -117,7 +117,7 @@ $background-colors: map-merge(
     "success": $supporting-green,
     "info":    $supporting-blue,
     "warning": $supporting-yellow,
-    "danger":  $purple
+    "danger":  $danger
```

Comment:
Going this way, you should probably use functional colors above too instead of supporting ones, isn't it?
Or does inconsistency acceptable here?

Definitely a topic to address wth the brand team, giving a red supporting color instead of the functional one, IMHO.

Reply:
Created an issue to keep track of this